README.md

```diff
@@ -31,3 +31,11 @@ Before submitting a pull request run:
 ```
 npm test
 ```
+
+### Contributing
+
+You can start off with easy issues, you can find [here](https://github.com/mbebenita/WebAssemblyStudio/labels/good%20first%20issue)
```

Comment:
This can just simply say: "You can start off with easy issues" and "easy issues" can be a link.